### PR TITLE
feat: Option to disable thumbnails.

### LINKED ISSUE #439: Option to turn off the newest style change
The latest update turned on a new multi-line tab style with website thumbnails:

screen shot 2016-07-22 at 11 06 15

I personally don't think it looks very good, in fact I much preferred the old style. 

In addition to the looks, which is of course subjective, I also find it a bit too "cluttered," in that it makes it more difficult to visually "skim" through the list to find the tab you're looking for. Finally, I think it wastes too much space. 

When I saw the change, I looked for an option to turn it off, and I was kind of disappointed there was none (not even a hidden setting in about:config.)

But I also understand this is project is a beta where experiments are exactly supposed to happen, and that settings take time and effort to code and test. So I'll just downgrade my plugin to the previous version for the time being, because for me this change was a net loss in usability. 

Hope this helps. Keep up the good work!

### PR DESCRIPTION
**Pull request by Popog**
Fixes #439

### CODE REVIEW

**Comment by Popog** [comment]
Right, so like I was saying, Creating this as a hidden option means turning it on when you're ready is just removing `'hidden': true,`.

File: package.json [hunk 1]
```diff
@@ -57,6 +57,28 @@
       "description": "New tabs are opened at the top",
       "type": "bool",
       "value": false
+    },
+    {
+      "name": "largetabs",
+      "hidden": true,
+      "title": "Large Tabs",
+      "description": "Tabs are displayed large with thumbnail",
```
Comment (bwinton) [inline package.json]: Can we change this to "Tabs display thumbnails when there is enough room."?

**Comment by bwinton** [comment]
Yeah, that seems reasonable.  Also (and I understand this isn't caused by your changes) I just noticed that if you have the tabs in pinned mode they won't switch to small mode after you create a lot of them.  Since this code is related to that, would you mind taking a stab a fixing that bug too?  🙂

**Comment by bwinton** [comment]
(Also, I hadn't seen the hidden option before.  That's pretty cool!  😄)

File: package.json (hunk 1 above)
Comment (Popog) [inline package.json]: That's only for the "Auto" setting. I also included an "Always" setting for having the tabs always display thumbnails.

Comment (bwinton) [inline package.json]: Hmm, okay, then how about "Display thumbnails in tabs", and change "Auto" to "When there's room"…

Comment (Popog) [inline package.json]: Sounds good to me.

**Comment by Popog** [comment]
Sure, no idea what the issue number for that is, but switching `hidden=false` to `:not([hidden=true])` seems like it fixed it.

**Pull request merged**